### ISSUE TITLE
Code cannot be read and edited using a screen reader in Android web version of VS code.

### ISSUE THREAD

**Issue by rahulbhai9** [microsoft/vscode#198612]
Does this issue occur when all extensions are disabled?: Yes/No

- VS Code Version: web
- OS Version: android 10

Steps to Reproduce:

1. Turn on TalkBack from settings >accessibility
2. open any file in web version of VS code.
3. Try to read the code inside the file with screen reader. Whenever the edit box is focused, the screen reader only announces name of the file with the edit box and does not announces any code written in the file. Hence the code cannot be edited as well. Also after typing some code if enter is pressed (in the mobile keyboard)the edit box becomes blank for the screen reader and it is not possible to read previous lines of that file.

**Comment by meganrogge** [comment]
@isidorn any history / discussion around TalkBack?

**Comment by rahulbhai9** [comment]
As far as I know, it looks like vsco does not use standard input box instead it is using some kind of custom implementation of it. In my experience custom input boxes does not work well with the TalkBack. That's why most of the code editors are inaccessible by design. One quick fix can be to allow a user to switch to standard input box for editing the code. It is acceptable as of now if auto complete encode highlighting does not work in that mode.

**Comment by isidorn** [comment]
@meganrogge no prior discussions on TalkBack. We do not officially support it. I suggest to keep this issue open for more feedback and no action for now. Thank you

**Comment by rahulbhai9** [comment]
Pleas, help. I would like to find the code which is responsible for running vs code on web so that I can play with it.
